systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterIT.java

```diff
@@ -168,4 +170,10 @@ public String consumeMessages(String clusterName, String topic, int groupID, int
                             "-kafka:9092 --topic " + topic + " --group-id " + groupID + " & sleep "
                             + timeout + "; kill %1").out();
     }
+
+    void checkErrorsInLogCC() {
```

Comment:
`assertNoCcErrorsLogged()` might be a better name – it is ultimately an assertion.